wagtail/admin/templates/wagtailadmin/shared/page_status_tag.html

```diff
@@ -1,5 +1,11 @@
+{% load i18n %}
 {% if page.live %}
-    <a href="{{ page.url }}" target="_blank" rel="noopener noreferrer" class="status-tag primary">{{ page.status_string }}</a>
+    <a href="{{ page.url }}" target="_blank" rel="noopener noreferrer" class="status-tag primary" title="Visit the live page">
```

Comment:
`Visit the live page` should be translatable.

Reply:
Resolved in https://github.com/wagtail/wagtail/pull/5372/commits/394420357205ea8e4e36ee4fa806d8387aea31c0